pyproject.toml

```
@@ -152,6 +153,7 @@ dask = ["dask", "regex", "packaging"]
 datafusion = ["datafusion"]
 druid = ["pydruid"]
 duckdb = ["duckdb"]
+e6data = ["e6data-python-connector","pymysql"]
```

```suggestion
e6data = ["e6data-python-connector"]
```

Is `pymysql` actually required? It doesn't seem to be used anywhere.